countdown/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Timer</title>
+        <link rel="stylesheet" type="text/css" href="style.css"/>
+    </head>
+    <body>
+        <input id="hours"> hours
+        <input id ="minutes" placeholder="25"> minutes
+        <input id="seconds" placeholder="0"> seconds
+        <button onclick="manageInterval()">Start</button>
+        <button onclick="pauseInterval()">Pause</button>
+        <p id="time"></p>
```

Comment:
for now this is fine but I'd encourage you to think about how to organize into parts that are different/similar. Like the inputs should be contained together. The buttons should live inside another container.